cms/envs/bok_choy.auth.json

```diff
@@ -45,6 +45,7 @@
             "ENGINE": "xmodule.modulestore.mixed.MixedModuleStore",
             "OPTIONS": {
                 "mappings": {},
+                // TODO: should split be specified here?
                 "stores": [
```

Comment:
@jzoldak bok choy tests run with the split modulestore by default. This is done through the paver commands.

But in the bok_choy.auth.json files (also see 2nd file below), split isn't even listed as one of the modulestores.

Should this configuration be updated to list split? Or deleted if it is not used?